NEWS

```diff
@@ -11,6 +11,172 @@
 ### Features:
 ### Bugfixes:
 
+## 1.13.0 (May 17, 2022)
+#### Features
+##### Core
+* Added new objects to VFS: ep local and remote address (e5dff69d5), number of total/failed EP creations (b970b7302), ep destroys and failed eps (6bf7f7d2f)
```

Comment:
is it a new approach that we specify sha?

Reply:
No, just to structurize, I can delete them before merge if it's not needed